instructor/templates/instructor/wb_band_intensity.html

```diff
@@ -34,11 +34,11 @@
                     <div class="scb_ab_s_col_width_sample">
                         <div class="block-ellipsis">
                         {{ form.instance.strain_protocol.strain.name }},
-                        {{ form.instance.strain_protocol.treatment.drug.name }}{% if variables.has_concentration %},
+                        {{ form.instance.strain_protocol.treatment.drug.name }}{% if variables.has_concentration and form.instance.strain_protocol.treatment.drug.concentration %},
```

Comment:
The expression here checks if `drug.concentration` is falsy, not strictly that it's not none, so if you had a value `0` it would not be printed. Can you add ` != None` to the three expressions here to guard against this?